pontoon/contributors/templates/contributors/settings.html

```diff
@@ -64,6 +64,18 @@ <h3>Personal information</h3>
         </div>
       </section>
 
+   <section class="appearance" >
+        <h3>Appearance</h3>
+        <div class="field">
+            <p class="help">Choose if Pontoon’s appearance should be dark, light, or follow your system’s settings</a></p>
+        </div>
+        <span class="toggle-button">
+            <button type="button" class="dark active" title="Use a dark theme in Pontoon"><i class="icon far fa-moon"></i>Dark theme</button>
+            <button type="button" class="light" title="Use a light theme in Pontoon"><i class="icon fa fa-sun"></i>Light theme</button>
+            <button type="button" class="system" title="Use a theme in Pontoon that matches your system settings"><i class="icon fa fa-laptop"></i>Match system</button>
```

Comment:
Please make sure the text matches the spec. We dropped "Pontoon". Please also do so with the other two themes.